float64 roll
float64 pitch
float64 yaw
float32 init_yaw
float32 tf_yaw
float32 current_drift
float32 average_drift
float32 no_drift_yaw